app/views/staff/activity_forms/country.html.haml

```diff
@@ -1,2 +1,2 @@
 = render layout: "wrapper" do |f|
-  = f.govuk_collection_select :recipient_region, region_select_options, :code, :name, options: { selected: f.object.recipient_region || region_select_options.first.code }, label: { tag: 'h1', size: 'xl' }
+  = f.govuk_collection_select :recipient_country, country_select_options, :code, :name, options: { selected: f.object.recipient_country || country_select_options.first.code}, label: { size: "xl", tag: "h1" }
```

Comment:
I _think_ `code` is an empty string by default. Is that intended?

Reply:
Yes, country is a select box so I put in a empty value as the default that fails the validation.

Reply:
Would it be clearer to set it to `nil`, since that's what you're really representing?

Reply:
I agree it would be slightly more intuitive if this was set to nil since we are unlikely to ever set a default country.

Reply:
That said, I don't think it should block this work. We'll have 2 more iterations planned to this country form page.